drivers/bridge/bridge.go

```diff
@@ -675,6 +676,7 @@ func (d *driver) createNetwork(config *networkConfiguration) error {
 	if !bridgeAlreadyExists {
 		bridgeSetup.queueStep(setupDevice)
 	}
+	config.ifaceManaged = !config.DefaultBridge && !bridgeAlreadyExists
```

Comment:
I prefer as it was before, where `config.ifaceManaged = true` inside the if block.
But changing the if check below.

Because I think we should mark the default network bridge with !ifaceManaged just so we have cleaner diffs.
The default network bridge is in fact managed by libnetwork.